dids/module.md

```diff
@@ -58,15 +46,137 @@ val opts = CreateDidDhtOptions(
 val did = DidDht.create(keyManager, opts)
 ```
 
-### Resolving a DID
+## Resolution
+
+### Resolve a DID DHT
+
+```kotlin
+val did = DidDht.resolve("did:dht:uAAAAB")
```

Comment:
is this a real life did:dht?

Reply:
😮

Reply:
Fixed in https://github.com/TBD54566975/web5-kt/pull/142/commits/4a6d11257e6e9087d1f9c59d4b41f27ac08afbd3

Note that it's currently borked, but will fix soon.